### PR TITLE
fix invalid branch due to double / when no image tag

### PR DESCRIPTION
**Pull request by fredericfran-gds**

### CODE REVIEW

File: charts/argo-workflows/scripts/set-automatic-deploys-enabled.sh [hunk 1]
```diff
@@ -1,7 +1,11 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-BRANCH="update-image-tag/${REPO_NAME}/${ENVIRONMENT}/${IMAGE_TAG}/${AUTOMATIC_DEPLOYS_ENABLED}"
+BRANCH="update-image-tag/${REPO_NAME}/${ENVIRONMENT}/${AUTOMATIC_DEPLOYS_ENABLED}"
+if [[ -z "${IMAGE_TAG}" ]]; then
```
Comment (sengi) [inline charts/argo-workflows/scripts/set-automatic-deploys-enabled.sh:5]: Is the condition the right way around?

Comment (fredericfran-gds) [reply charts/argo-workflows/scripts/set-automatic-deploys-enabled.sh:5]: nah, fixed

File: charts/argo-workflows/scripts/set-automatic-deploys-enabled.sh [hunk 2]
```diff
@@ -21,7 +25,7 @@ commit() {
 }
 
 update_image_tag() {
-  if [[ -z "${IMAGE_TAG}" ]]; then
+  if [[ -n "${IMAGE_TAG}" ]]; then
```
Comment (sengi) [inline charts/argo-workflows/scripts/set-automatic-deploys-enabled.sh:28]: Good catch!

**Review by sengi — APPROVED** [review]
Thanks!

**Pull request merged**